app/lib/govuk/markdown_renderer.rb

```diff
@@ -25,14 +25,18 @@ def list(content, list_type)
       end
     end
 
-    def link(link, title, content)
+    def link(link, _title, content)
       <<~HTML
-        <a href="#{link}" title="#{title}" class="govuk-link">
+        <a href="#{link}" class="govuk-link">
```

Comment:
I think the line break here is causing an extra space to appear in the markup:

This markdown:
![Screen Shot 2019-05-30 at 16 31 04](https://user-images.githubusercontent.com/319055/58644088-7781b400-82f8-11e9-9d31-30fa2d386672.png)

Comes with a space after it:
![Screen Shot 2019-05-30 at 16 30 45](https://user-images.githubusercontent.com/319055/58644090-7781b400-82f8-11e9-80d4-1c7900038d85.png)